parsers/AR.py

```diff
@@ -621,7 +625,7 @@ def get_hydro(session=None):
     full_table = []
     
 
-    for pagenumber in range(1,2,1):
+    for pagenumber in range(1,5,1):
```

Comment:
Yes we can switch it to a while loop with a flag variable to improve future robustness.